xml/System.Data.Common/DbDataReader.xml

```diff
@@ -1120,26 +1103,9 @@ private static void GetDataTypes(String connectionString) {
       </Parameters>
       <Docs>
         <typeparam name="T">The type of the value to be returned.</typeparam>
-        <param name="ordinal">The type of the value to be returned.</param>
-        <summary>Asynchronously gets the value of the specified column as a type.</summary>
-        <returns>The type of the value to be returned.</returns>
-        <remarks>
-          <format type="text/markdown"><![CDATA[  
-  
-## Remarks  
- `T` can be one of the following types:  
-  
-|||||  
-|-|-|-|-|  
-|Boolean|Byte|Char|DateTime|  
```

Comment:
Supported types is provider specific.

Reply:
Yeah, this specific fragment should be moved to SqlClient...